src/components/BaseComponents/UserPopup/UserPopup.js

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import "../../../styles/components/popup.scss";
+
+const UserPopup = ({ popup, setPopup, user }) => {
+    console.log(user);
+    return (
+        <>
+            {popup ? (
+                <div className="popup">
+                    <div
+                        className="popup__close"
+                        onClick={() => setPopup(false)}
+                    >
+                        x
+                    </div>
```

Comment:
Would be super cool to implement a complex solution. We should close the modal by clicking outside the box. The easiest way - a third-party library, but you can learn something by implementing it on your own :)